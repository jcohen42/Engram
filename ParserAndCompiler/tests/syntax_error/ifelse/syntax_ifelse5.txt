Begin program:
	Set x equal to 76.
	Set y equal to 76.
	If x is equal to y:
		Output "Equal 1".
	End if.
	End if.
	Set y equal to -38.
	If x is equal to y times -2:
		Output "Equal 2".
	End if.
End program.