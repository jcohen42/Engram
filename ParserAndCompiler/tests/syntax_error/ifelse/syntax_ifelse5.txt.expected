Syntax error encountered on line 7!